ign_rviz_plugins/src/rviz/plugins/laser_scan_display.cpp

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2020 Open Source Robotics Foundation, Inc.
```

Comment:
This class should be replaced with the LidarVisual when it's ready. Can you create a ticket for this?

Reply:
Created a new ticket #19